types/fuelux/index.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for jQuery jqgrid Plugin 1.3
```

Comment:
Should read `// Type definitions for fuelux 3.16`? (3.16 is the newest version of fuelux according to `npm info fuelux`)